docs/public/index.css

```diff
@@ -78,19 +79,9 @@ h5 {
   font-size: 0.8rem;
 }
 
-@media (min-width: 50em) {
-  h1 {
-    font-size: 3.25rem;
-    font-weight: 800;
-  }
-
-  h2 {
-    font-size: 2rem;
-  }
-
-  h3 {
-    font-size: 1.25rem;
-  }
+p {
+  letter-spacing: -.005625rem;
```

Comment:
general suggestion: letter-spacing should always be `em`, not `rem`, so it scales properly.

For this change, I’m -1 on adjusting letter-spacing for paragraph, because that should be the responsibility of the typeface, and it will a) not work if a different typeface is used, or b) you’re compensating for some browser rendering thing which is not only typeface-specific but browser-specific, too. If you’re using a half-decent typeface (which we are) then the type designer has probably optimized this already.

IMO best time to adjust letter-spacing is when you’re fine-tuning really large headings, or kerning a fixed graphic.

Reply:
awesome, love this point. I bow down to the typography gurus